packages/fe-container/src/components/top-bar/TopBar.test.tsx

```diff
@@ -39,11 +40,39 @@ describe('TopBar tests', function () {
   it('Closed TopBar is opening', () => {
     const mockBurgerClick = jest.fn(isToggled => {
     })
+    const theming = {
```

Comment:
if theming is the same between all the tests, please extract it at describe level